CHANGELOG.md

```diff
@@ -1,29 +1,31 @@
-## [0.4.2](https://github.com/adfinis-sygroup/ember-simple-auth-oidc/compare/v0.4.1...v0.4.2) (2019-09-09)
+## [0.4.3](https://github.com/adfinis-sygroup/ember-simple-auth-oidc/compare/v0.4.2...v0.4.3) (2019-10-04)
+
+### Changes
 
+- **dependencies:** update dependencies
+
+## [0.4.2](https://github.com/adfinis-sygroup/ember-simple-auth-oidc/compare/v0.4.1...v0.4.2) (2019-09-09)
 
 ### Bug Fixes
 
-* **authenticator:** await successful retry before setting the session ([18b9c1f](https://github.com/adfinis-sygroup/ember-simple-auth-oidc/commit/18b9c1f))
+- **authenticator:** await successful retry before setting the session ([18b9c1f](https://github.com/adfinis-sygroup/ember-simple-auth-oidc/commit/18b9c1f))
```

Comment:
Please don't alter the markdown you didn't really touch

Reply:
Reverted